templates/named.conf.erb

```diff
@@ -21,6 +21,9 @@ options {
 <%-   end -%>
 	};
 <%- end -%>
+<%- if @forward != '' -%>
```

Comment:
Does `undef == ''`? I'm trying to work out what happens when this parameter is not set and retains its default value of `undef`, but puppet's types have always been unlike anything else.